packages/shared/src/components/GalleryProccessedText/GalleryTextElementParser.ts

```diff
@@ -51,12 +53,15 @@ export function getMentionElements(
     );
   }
 
-  const mentions: GalleryTextElementParserMentionsFragment$data[] = [];
+  let mentions: GalleryTextElementParserMentionsFragment$data[] = [];
 
   mentionRefs.forEach((mentionRef) => {
     mentions.push(fetchMention(mentionRef));
   });
 
+  // Remove duplicate mentions
+  mentions = uniqWith(mentions, isEqual);
```

Comment:
do you know what causes duplicate mentions? I wonder if there is a separate bug that causes duplicate mentions in the first place that we should also fix

Reply:
@kaitoo1 Not sure 🤔 I'm currently to find what cause the issue. There must be something that triggered it

Reply:
replied on slack!